### PR TITLE
Basic Springboard Getting Started (with Intellij) tutorial

### PR DESCRIPTION
**Pull request by hevb**
https://3.basecamp.com/4506496/buckets/16968057/todos/2689389855

### CODE REVIEW

File: docs/docs/tutorials/gettingstarted.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+using another IDE, let us know and we can add it!
+
+This tutorial intentionally goes step-by-step through common errors or mistakes someone might make while trying to get the project running.
+If you are an experienced developer- some of it might seem obvious and unnecessary to you.
+
+## Pulling the Project form GitHub
+This might be an obvious one- but use the git clone command to pull down the Springboard Project.
+
+```bash
+git clone git@github.com:HappyValleyIO/springboard.git rvroger
```
Comment (stepbeekio) [inline docs/docs/tutorials/gettingstarted.md:25]: Should we change the name of the project?

File: docs/docs/tutorials/gettingstarted.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+configured some test database properties. 
+ 
+ [![Initial DEV Application Properties File](images/getting-started-dev-application-properties.png)](images/getting-started-dev-application-properties.png)
+ 
+Since the DEV application properties file contains everything we need to run a local development instance of our 
+application, we need to make sure that this is the file being picked up when we run. Let's edit the Run Configuration 
+for SpringboardApplication so that it uses the DEV spring profile. Since we tried  to run this already- 
+IntelliJ has a Run Configuration setup for us if we "Edit Configurations" from the run configuration menu.
+  
+We need to add an environment variable:
```
Comment (stepbeekio) [inline docs/docs/tutorials/gettingstarted.md:84]: You can also configure the profile directly in intellij.

File: docs/docs/tutorials/gettingstarted.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+[![Website Landing Page](images/updated-variables.png)](images/updated-variables.png)
+
+Hit 'CTRL + F9' to rebuild the project - and watch the browser change from the Springboard maroon/red to our new primary blue color.
+
+[![Website Landing Page](images/updated-primary-variable-landing-page.png)](images/updated-primary-variable-landing-page.png)
+
+That's just a starting point- but it lets you see how easily you can start to customize Springboard to get your 
+application moving quickly. Changing something as simple as the primary project color starts to make the project feel 
+like *your* application, not just the template.
```
Comment (stepbeekio) [inline docs/docs/tutorials/gettingstarted.md:197]: Maybe add a next steps section? 

```
Now that you've got things going, why not learn how to write a Cypress test or how to deploy to production with heroku?

**Review by stepbeekio — APPROVED** [review]
Looks great! Minor changes needed, but I'm happy for you to self-merge.

File: docs/docs/tutorials/gettingstarted.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
+---
+title: Getting Started with Springboard
+summary: A walkthrough of how to pull down the Springboard project, get it running for the first time in Intellij and 
+start customizing a few pieces of the project.
```
Comment (stepbeekio) [inline docs/docs/tutorials/gettingstarted.md:4]: Ah! I think the newline in here is breaking the change:

https://happyvalleyio.github.io/springboard/tutorials/gettingstarted/

File: docs/docs/tutorials/gettingstarted.md (hunk 1 above)
Comment (hevb) [reply docs/docs/tutorials/gettingstarted.md:25]: It was in all the screenshots I took- so I didn't know? Do you want me to redo them? It won't take too long

File: docs/docs/tutorials/gettingstarted.md (hunk 2 above)
Comment (hevb) [reply docs/docs/tutorials/gettingstarted.md:84]: How?

File: docs/docs/tutorials/gettingstarted.md (hunk 3 above)
Comment (hevb) [reply docs/docs/tutorials/gettingstarted.md]: done.

File: docs/docs/tutorials/gettingstarted.md (hunk 4 above)
Comment (hevb) [reply docs/docs/tutorials/gettingstarted.md]: I have fixed this!

File: docs/docs/tutorials/gettingstarted.md (hunk 2 above)
Comment (stepbeekio) [reply docs/docs/tutorials/gettingstarted.md:83]: Scroll down in the Run Configuration window for a Spring run and you'll see it.

File: docs/docs/tutorials/gettingstarted.md (hunk 1 above)
Comment (stepbeekio) [reply docs/docs/tutorials/gettingstarted.md:24]: Nah, it'll be fine.

**Comment by hevb** [comment]
Merged in master to pick up the semaphore fixes

**Pull request merged**